test/meta-buildpacks/java-function/build.sh

```diff
@@ -1 +0,0 @@
-../../../common/meta-build.sh
```

Comment:
(Optional) The `common/meta-build.sh` script appears to be unused after this change (though you may be wanting to keep it for adhoc local testing; up to you)

Reply:
Removed in 8a7bc86